### PR TITLE
A Few Quick Accessibility Improvements

### PR DESCRIPTION
**Pull request by pbailie**
Quick-fixes for a few `label` and `aria-labelledby` accessibility errors.

Note that this does **not** correct accessibility issues involving jump menu ~~or misuse of HTML for styling~~ as such corrections could have an impact on the UI.

### CODE REVIEW

File: site/app/templates/admin/Extensions.twig [hunk 1]
```diff
@@ -37,22 +37,22 @@
...
         </fieldset>
         <fieldset>
-            <legend class="option-title">Multiple Student Entry Via CSV Upload</legend>
-            <label for="csv_upload">
+            <legend class="option-title" id="csv_upload_title">Multiple Student Entry Via CSV Upload</legend>
+            <label for="csv_upload" id="csv_upload_format">
                 Do not use column headers. CSV must be of the following form: student_id, gradeable_id, days_of_extension
             </label>
-            <input type="file" name="csv_upload" aria-labelledby="csv_upload_title csv_upload_format" onchange="updateHomeworkExtension()">
+            <input type="file" name="csv_upload" id="csv_upload" aria-labelledby="csv_upload_title csv_upload_format" onchange="updateHomeworkExtension()">
```
Comment (MasterOdin) [inline site/app/templates/admin/Extensions.twig]: Could you please update the `id` and `aria-labelledby` to use `kebab-case` here?

File: site/app/templates/course/UploadCourseMaterialsForm.twig [hunk 2]
```diff
@@ -29,7 +29,7 @@
         <div id="upload-boxes">
             {# upload1 and input_file1 required for drag-and-drop.js #}
             <div id="upload1" class="file-input">
-                <label class="label" for="input-file">"Drag your file(s) here or click to open file browser"</label>
+                <label class="label" for="input_file1">"Drag your file(s) here or click to open file browser"</label>
                 <input type="file" name="files" id="input_file1" onchange="addFilesFromInput(1,false)" multiple />
```
Comment (MasterOdin) [inline site/app/templates/course/UploadCourseMaterialsForm.twig]: could you update this to be `kebab-case` here please?

File: site/app/templates/admin/Extensions.twig (hunk 1 above)
Comment (pbailie) [reply site/app/templates/admin/Extensions.twig]: done

File: site/app/templates/course/UploadCourseMaterialsForm.twig (hunk 2 above)
Comment (pbailie) [reply site/app/templates/course/UploadCourseMaterialsForm.twig]: done

**Pull request merged**